lambdatimes/src/components/Content/Cards.js

```diff
@@ -1,15 +1,29 @@
 import React, { Component } from 'react';
 import Card from './Card';
+import PropTypes  from "prop-types";
 
 const Cards = props => {
   return (
     <div className="cards-container">
-      {/* Using the cards prop, map over the list creating a 
+      {/* Using the cards prop, map over the list creating a
           new Card component for each passing the card as the only prop*/}
```

Comment:
Don't forget to add a key to the Card when we map it to get rid of the console warnings